### PR TITLE
added parameter for color argument to world.cube method

### PR DESCRIPTION
**Pull request by friscodelrosario**
The world.cube method accepts an argument for color, so if the user wants to place one block of colored wool, then world.cube(x,y,z,x,y,z,wool,color).

### CODE REVIEW

File: lib/world.rb [hunk 1]
```diff
@@ -14,8 +14,8 @@ def block(x,y,z,block)
     @api.send("world.setBlock(#{x},#{y},#{z},#{block.as_parameter_list})")
   end
 
-  def cube(x1,y1,z1,x2,y2,z2,block)
-    @api.send("world.setBlocks(#{x1},#{y1},#{z1},#{x2},#{y2},#{z2},#{block.as_parameter_list})")
+  def cube(x1,y1,z1,x2,y2,z2,block,color)
+    @api.send("world.setBlocks(#{x1},#{y1},#{z1},#{x2},#{y2},#{z2},#{block.as_parameter_list},#{color})")
```
Comment (friscodelrosario) [inline lib/world.rb]: Oh yes, you are right.

Comment (friscodelrosario) [inline lib/world.rb]: How about a wool() method?

**Comment by nhajratw** [comment]
Hmm .. i just went back and looked at the code (it's been a couple years ;-) ) --- Did you see the Wool class? You can create wool blocks with a color. You can then pass that wool block to the cube method, etc...

Is this different that what you're trying to do?

**Comment by friscodelrosario** [comment]
Aha. Thank you!

**Comment by nhajratw** [comment]
FYI I updated the README to contain a wool example.

**Pull request closed without merging**